Library/Formula/macbrush.rb

```diff
@@ -0,0 +1,20 @@
+class Macbrush < Formula
+  desc "Keeping OSX directories clear of temporary files"
+  homepage "http://nils-tekampe.github.io/MacBrush"
```

Comment:
`https://nils-tekampe.github.io/macbrush/`. Should be both `https` and lowercase, otherwise it 404s.